special-interest-groups/observability-tag/README.md

```diff
@@ -0,0 +1,24 @@
+# Observability TAG
```

Comment:
minor: we renamed TAGs but the content is under the `special-interest-groups` dir. Perhaps renaming that also would be needed down the line

Reply:
There is an open issue about this topic: #49